src/core/src/pass/manager.cpp

```diff
@@ -36,35 +39,73 @@ PerfCounters& perf_counters() {
 #endif  // ENABLE_PROFILING_ITT
 
 namespace {
-bool getenv_visualize_tracing() {
-    return ov::util::getenv_bool("OV_ENABLE_VISUALIZE_TRACING");
-}
-}  // namespace
 
-ov::pass::Manager::Manager() : m_pass_config(std::make_shared<PassConfig>()), m_visualize(getenv_visualize_tracing()) {}
+/**
+ * @brief EnvVar gets the environment variable value by name.
+ * It tries to interpret the value as boolean, if it fails then
+ * the original string value is stored. This behavior helps us to reduce the number
+ * of the additional env variables.
+ *
+ * Example of usage:
+ * if OV_ENABLE_PROFILE_PASS is true, it enables console output.
+ * if OV_ENABLE_PROFILE_PASS contains a path to file (string), the out logs
+ * will be re-directed to the file.
+ */
+class EnvVar {
+public:
+    explicit EnvVar(const std::string& var) {
+        const auto& val = ov::util::getenv_string(var.c_str());
+        std::set<std::string> off = {"0", "false", "off"};
+        std::set<std::string> on = {"1", "true", "on"};
 
-ov::pass::Manager::~Manager() = default;
+        const auto& val_lower = ov::util::to_lower(var);
+        if (off.count(val_lower)) {
+            m_is_bool = true;
+        } else if (on.count(val_lower)) {
+            m_is_bool = true;
+            b_value = true;
+        } else {
+            s_value = val;
+        }
+    }
 
-ov::pass::Manager::Manager(std::shared_ptr<ov::pass::PassConfig> pass_config)
-    : m_pass_config(std::move(pass_config)),
-      m_visualize(getenv_visualize_tracing()) {}
+    /**
+     * @brief This ctor helps to activate/deactivate EnvVar from the code.
+     */
+    explicit EnvVar(const std::string& var, bool activate) {
+        m_is_bool = true;
+        b_value = activate;
+    }
 
-void ov::pass::Manager::set_per_pass_validation(bool new_state) {
-    m_per_pass_validation = new_state;
-}
+    bool is_enabled() const {
+        return b_value || !s_value.empty();
+    }
+
+    bool is_bool() const {
+        return m_is_bool;
+    }
+
+    std::string get_str() const {
+        return s_value;
+    }
+
+private:
+    bool m_is_bool = false;
+    bool b_value = false;
+    std::string s_value;
```

Comment:
```suggestion
    std::string s_value{};
```
Init all or none members.

Reply:
a default ctor of std::string will init it to empty string, we don't need to use {} here